### PR TITLE
data : rickmondal-2004

### PR DESCRIPTION
**Pull request by rickmondal-2004**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/8378"></a>

### CODE REVIEW

File: data/rickmondal_2004 [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name" : "Rick Mondal",
+  "displayStatsPublic" : true,
+  "type" : "personal",
```
Comment (SaraJaoude) [inline data/rickmondal_2004:4]: no longer required:
```suggestion
```

File: data/rickmondal_2004 [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  "bio" : "Student | Web Developer | Graphic Designer",
+  "avatar" : "https://github.com/rickmondal-2004.png"
```
Comment (SaraJaoude) [inline data/rickmondal_2004:6]: no longer required:
```suggestion
```

File: data/rickmondal_2004 [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    {
+      "name" : "Folow my personal blog",
+      "url" : "https://rick2004.hashnode.dev/",
+      "icon" : "SiHashnode"
+    },
+     {
+      "name" : "follow me on Instagram",
+      "url" : "https://www.instagram.com/rickmondal_2004",
+      "icon" : "FaInstagram"
+    },
```
Comment (SaraJaoude) [inline data/rickmondal_2004:34]: remove comma:
```suggestion
    }
```

**Review by SaraJaoude — CHANGES_REQUESTED** [review]
Please make sure your file name matches your GitHub username exactly (it is case sensitive). Instead of `data/rickmondal_2004`   it should be `data/rickmondal-2004.json` (essentially you have `_` instead of a `-`).

I would suggest you look at our QuickStart Guide (especially section 7) in Docs: https://linkfree.io/docs/quickstart-json to help you update this Pull Request.

**Review by rickmondal-2004 — COMMENTED** [review]
Accept pull requiest

**Review by Sindhuinti — CHANGES_REQUESTED** [review]
Hey @rickmondal-2004 inorder to accept your PR please make the requested changes
file name should be `data/rickmondal-2004.json`

**Comment by SaraJaoude** [comment]
> Accept pull requiest

@rickmondal-2004 this type of behaviour is not tolerated in the EddieHub community. Maintainers cannot merge your PR until the requested changes have been made.

**Comment by SaraJaoude** [comment]
We look forward to seeing your Profile on LinkFree, but as there is no recent activity from you to make the changes suggested I am going to close this pull request.

We hope you will raise a new pull request to create your Profile and when you do, you might want to take a look at the following Docs which will help with the suggested changes the maintainers have proposed:

Quickstart: https://linkfree.io/docs/quickstart-json

Editing your Profile: https://linkfree.io/docs/how-to-guides/editing-json

You can also create your Profile using Forms: https://linkfree.io/docs/quickstart-forms

**Pull request closed without merging**